### PR TITLE
Adds a new snail mob

### PR DESCRIPTION
**Pull request by cuackles**
# Document the changes in your pull request

adds a new snail mob so ian can bus with it and stuff, this compiled but i havent tested it, but i have faith it works
image

# Wiki Documentation

add to the list of gold stabilized pets if it exists
# Changelog

:cl:  
rscadd: Added new simple mob   (snail)    
imageadd: added some icons and images  of said simple mob (snail)
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
when snails

**Comment by Sniblet** [comment]
Gary!!

File: code/modules/mob/living/simple_animal/friendly/snail.dm [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	speak_emote = list("gurgles")
+	emote_hear = list("gurgles.")
+	emote_see = list("gurgles.")
+	speak_chance = 1
+	turns_per_move = 5
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 1)
+	response_help  = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm   = "stomps"
+	stop_automated_movement = 1
```
Comment (nmajask) [inline code/modules/mob/living/simple_animal/friendly/snail.dm]:
```suggestion
	stop_automated_movement = TRUE
```

**Comment by JayBaez** [comment]
make a borg skin

**Comment by cuackles** [comment]
> make a borg skin
> 

Make this a Borg skin or make a Borg skin based on this ? Cus we already have snail Janiborg

**Comment by JamieD1** [comment]
You totally _shouldnt_ make a pet named gary and place them somewhere.

**Review by nesisknesis — APPROVED** [review]

**Comment by Mosslick** [comment]
Cringe

**Comment by JamieD1** [comment]
> You totally _shouldnt_ make a pet named gary and place them somewhere.

How we doing with my totally not good idea.

**Comment by cuackles** [comment]
> > You totally _shouldnt_ make a pet named gary and place them somewhere.
> 
> How we doing with my totally not good idea.

Oh i was gonna wait till it was merged and then add it to the map but i can try to get it all on the same pr

File: code/modules/mob/living/simple_animal/friendly/snail.dm [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/mob/living/simple_animal/pet/snail
+	name = "snail"
+	desc = "Snails are mollusks, putting them in the same category as oysters, clams, and mussels.
+	"
```
Comment (nmajask) [inline code/modules/mob/living/simple_animal/friendly/snail.dm]:
```suggestion
	desc = "Snails are mollusks, putting them in the same category as oysters, clams, and mussels."
```
Move this to the end of line 3

**Review by nmajask — DISMISSED** [review]

File: code/modules/mob/living/simple_animal/friendly/snail.dm (hunk 1 above)
Comment (nmajask) [reply code/modules/mob/living/simple_animal/friendly/snail.dm]: Ignore this

**Comment by cuackles** [comment]
image
NOW WITH ACTUALY WORKING SNAILS

File: code/modules/mob/living/simple_animal/friendly/snail.dm [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	health = 50
+	maxHealth = 50
+	gold_core_spawnable = FRIENDLY_SPAWN
+
+/mob/living/simple_animal/pet/snail/gary
+	name = "Gary the snail"
+	desc = "Gary come home"
+	icon_state = "snail"
+	icon_living = "snail"
+	icon_dead = "snail_dead"
```
Comment (JamieD1) [inline code/modules/mob/living/simple_animal/friendly/snail.dm]:
```suggestion
```

**Review by JamieD1 — DISMISSED** [review]

File: code/modules/mob/living/simple_animal/friendly/snail.dm (hunk 3 above)
Comment (JamieD1) [reply code/modules/mob/living/simple_animal/friendly/snail.dm]: Unneeded as its the same as the parent.

**Comment by JamieD1** [comment]
@wejengin2

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**